code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -452,8 +452,8 @@
 	for(var/organ in contents)
 		var/obj/item/organ/O = organ
 		if(!istype(O))
-			return
-		O.applyOrganDamage(-repair_rate * delta_time)
+			continue
+		O.applyOrganDamage(-repair_rate * O.maxHealth * delta_time)
```

Comment:
Turn this into `for (var/obj/item/organ/organ in contents)` and remove the istype.